src/tests/ftest/ior/ior_intercept_messages.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import re
+import os
+from ior_test_base import IorTestBase
+
+
+class IorInterceptMessages(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description: Runs IOR with interception.
+
+       Look for messages provided the the library
+
+    :avocado: recursive
+    """
+
+    def test_ior_intercept_messages(self):
+        """Jira ID: DAOS-7647.
+
+        Test Description:
+            Purpose of this test is to run ior using dfuse with interception
+            library enabled and look for some debug messages provided
+            by the library.
+
+        Use case:
+            Run ior with dfuse flags set on debug
+            Look for interception library messages.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,small
+        :avocado: tags=daosio,dfuse,il
+        :avocado: tags=iorinterceptmessages
+        """
+        apis = self.params.get("ior_api", '/run/ior/iorflags/ssf/*')
```

Comment:
This line can be avoided by simply specifying api under `/run/ior/` in yaml file as follows:
```
ior:
    api: POSIX
```
And you can later access this value using self.ior_cmd.api.value